repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+        /haematology__doctor_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: haematology__doctor_
+          locale.en.name: Histopathology (doctor)
+        /immunology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: immunology
+          locale.en.name: Infectious diseases
+        /intensive_care_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: intensive_care_medicine
+          locale.en.name: Liaison psychiatry
+        /medical_microbiology_and_virology__doctor_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: medical_microbiology_and_virology__doctor_
+          locale.en.name: Medical microbiology and virology (doctor)
+        /medical_oncology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: medical_oncology
+          locale.en.name: Medical oncology
+        /medical_ophthalmology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: medical_ophthalmology
+          locale.en.name: Medical ophthalmology
+        /medical_psychotherapy:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: medical_psychotherapy
+          locale.en.name: Medical psychotherapy
+        /metabolic_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: metabolic_medicine
+          locale.en.name: Metabolic Medicine
+        /neurology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: neurology
+          locale.en.name: Neurology
+        /neurosurgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: neurosurgery
+          locale.en.name: Neurosurgery
+        /nuclear_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: nuclear_medicine
+          locale.en.name: Nuclear medicine
+        /obstetrics_and_gynaecology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: obstetrics_and_gynaecology
+          locale.en.name: Obstetrics and gynaecology
+        /occupational_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: occupational_medicine
+          locale.en.name: Occupational medicine
+        /old_age_psychiatry:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: old_age_psychiatry
+          locale.en.name: Old age psychiatry
+        /ophthalmology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: ophthalmology
+          locale.en.name: Ophthalmology
+        /oral_and_maxillofacial_surgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: oral_and_maxillofacial_surgery
+          locale.en.name: Oral and maxillofacial surgery
+        /otorhinolaryngology__ear,_nose_and_throat_surgery,_ent_:
```

Comment:
Can we remove the `,` from both the name and key ?

Reply:
Going through and changing all of these now